me.hyliu.fluentreader.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
Should be upstreamed.

Reply:
It's in upstream. It's generated by `electron-builder` and of course I've polished some fields.